config/samples/smartgateway_v2_smartgateway.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: smartgateway.infra.watch/v2
+kind: SmartGateway
+metadata:
+  name: smartgateway-sample
+spec:
+  applications:
+    - config: |
+        host: 127.0.0.1
+        port: 8081
+        withTimeStamp: true
+      name: prometheus
+  bridge:
+    enabled: false
+  handleErrors: true
+  logLevel: info
+  services:
+    - name: smartgateway-sample
+      ports:
+        - name: prom-http
+          port: 8083
```

Comment:
For the benefit of other reviewers... This looks wrong (8081 != 8083) but is actually correct since that endpoint is protected by the oauth-proxy on port 8083